deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/AgentJanitor.java

```diff
@@ -122,6 +122,7 @@ private boolean isHostStale(HostAgentBean hostAgentBean) {
         }
 
         if (janitorStartTime - hostAgentBean.getLast_update() >= absoluteThreshold) {
+            LOG.info("exceeded absolute stale threshold ({}) for host ({})", absoluteThreshold, hostAgentBean);
```

Comment:
@tylerwowen Do you think INFO is the correct log level here?

Reply:
Slightly prefer debug, but info is fine.

Reply:
I do too, wanted to hear what you thought. Changing it now